Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift

```diff
@@ -523,7 +523,8 @@ final class FileManagerTests : XCTestCase {
             XCTAssertFalse($0.fileExists(atPath: "does_not_exist"))
         }
     }
-    
+
+#if !os(Windows)
```

Comment:
Could we leave a comment about why this test isn't supported on Windows? (sometimes I find it helpful to put the if-def within the function and in the `os(Windows)` path `throw XCTSkip("Some explanation")` just so it's clear in the future that this is intentionally not supported (rather than the API just not being finished yet, for example) - not a requirement here but I think we should indicate the reasoning in some form)